### PR TITLE
Translating to bangla

### PR DESCRIPTION
**Pull request by MjAbuz**
- [x] Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you explained what your changes do, and why they add value to the Guides?

**Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

-----

### CODE REVIEW

**Review by MjAbuz — COMMENTED** [review]
reviewed security policy

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+# For most projects, this workflow file will not need changing; you simply need
```
Comment (gallexi) [inline .github/workflows/codeql-analysis.yml:1]: Is this file addition related to the translation? If not, can we remove it from this PR? Thanks!

**Review by gallexi — COMMENTED** [review]
The translation file looks good! Just left one question about the CodeQL file =]

**Comment by gallexi** [comment]
This is looking good! I've converted it to a draft while you're still working on it, but do feel free to mark it as "ready for review" when you're done!

**Comment by MjAbuz** [comment]
> This is looking good! I've converted it to a draft while you're still working on it, but do feel free to mark it as "ready for review" when you're done!

can i am to generate po/pot file from the directory ... is there any better 
way? @gallexi

**Comment by MjAbuz** [comment]
> This is looking good! I've converted it to a draft while you're still working on it, but do feel free to mark it as "ready for review" when you're done!

@gallexi 

It would be great to know if i can get po/pot file for translation before i do more, cause if yes to po/pot files than that will save a lot  of my time on type and that time i can use for proof reading.

**Comment by ramaazharywibisana** [comment]
> adding Bengali

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (ramaazharywibisana) [reply .github/workflows/codeql-analysis.yml:1]: @ramaazharywibisana 

>

**Review by ramaazharywibisana — CHANGES_REQUESTED** [review]

**Comment by gallexi** [comment]
> It would be great to know if i can get po/pot file for translation before i do more

Ah, alas we do not have the tooling set up for that, my apologies!

**Comment by lol983** [comment]
No you can not you need to just l stop and who is this tell me that

**Comment by ramaazharywibisana** [comment]
Im noylt understain


Pada tanggal Sab, 6 Mar 2021 06.27, lol983 <notifications@github.com>
menulis:

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (MikeMcQuaid) [inline .github/workflows/codeql-analysis.yml:1]: Please remove this file

**Comment by yrrahsilieb** [comment]
Files changed

**Review by luismedicina23 — APPROVED** [review]

**Review by luismedicina23 — APPROVED** [review]

**Review by luismedicina23 — APPROVED** [review]

**Review by Salim1989 — APPROVED** [review]

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (Salim1989) [reply .github/workflows/codeql-analysis.yml:1]: Как его удалить, для чего

**Review by Jsn2win — APPROVED** [review]
Looks ready to merge

**Comment by MjAbuz** [comment]
> > It would be great to know if i can get po/pot file for translation before i do more
> 
> Ah, alas we do not have the tooling set up for that, my apologies!

:( .. sorry I was away for a while ... and good news is we have our new family member at home .. a baby girl was born on 24th March ... hope to be back soon ...

**Review by reconsumeralization — CHANGES_REQUESTED** [review]
Looks good. Anyone have AI translator to confirm?

**Comment by noppaklaow34** [comment]
> * [x]  Have you followed the [contributing guidelines](https://github.com/github/opensource.guide/blob/HEAD/CONTRIBUTING.md)?
> * [x]  Have you explained what your changes do, and why they add value to the Guides?
> 
> **Please note: we will close your PR without comment if you do not check the boxes above and provide ALL requested information.**

**Review by reconsumeralization — APPROVED** [review]

**Review by jhalljorda — APPROVED** [review]

**Comment by gallexi** [comment]
> and good news is we have our new family member at home .. a baby girl was born on 24th March

Oh my goodness!! Congratulations to you and your family!!!

Absolutely no worries, take your time, enjoy!!

**Comment by jculver050** [comment]
Close

**Review by naataliie1 — APPROVED** [review]

**Review by katelynnashley95icloudcom — APPROVED** [review]

**Comment by noncash15** [comment]
MjAbuz:main

**Review by noncash15 — APPROVED** [review]

**Comment by MjAbuz** [comment]
I am back after a long break

File: _articles/bn/building-community.md [hunk 2]
```diff
@@ -0,0 +1,280 @@
+---
```
[the remark this answers is not in the document]
Comment (MjAbuz) [reply _articles/bn/building-community.md:1]: its done

[the remark this answers is not in the document]
Comment (itsbagpack) [reply _articles/bn/building-community.md:1]: Hi @MjAbuz, this file doesn't look to have been translated. Could you please remove it until it's been translated?

File: .github/workflows/codeql-analysis.yml (hunk 1 above)
Comment (itsbagpack) [reply .github/workflows/codeql-analysis.yml:1]: Please remove this file as it isn't relevant to the translation intended on this PR.

**Review by itsbagpack — COMMENTED** [review]
I've added some comments to existing conversations to remove files that are irrelevant/haven't been translated. We can re-review once those have been resolved. Thank you.
